### PR TITLE
SM-6845/new v3 documentation

### PR DESCRIPTION
**Pull request by lukeca362**
I have separated each step into the following commits:
- add v3 folders by duplicating v2 folders
- deleting all v1 folders [the biggest change in this PR]
- removing link to v1 documentation

### CODE REVIEW

File: docs/_includes/api-documentation/v3/versioning-and-release-management.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ This section aims to describe the approach taken by the Street Manager team in o
 ### API Versioning Approach
 {: .govuk-heading-m}
 
-The Street Manager API services will be versioned via the URL path, for example <code>api.sandbox.domain.com/v1/works/.../</code> versus <code>api.sandbox.domain.com/v2/works/.../</code>.  Initially, the Public Beta will focus on the <code>v1</code> major version of the codebase, with new major versions being introduced at a later date.
+The Street Manager API services are versioned via the URL path, for example <code>api.sandbox.domain.com/v1/work/.../</code> versus <code>api.sandbox.domain.com/latest/work/.../</code>. The UI will point at the <code>latest</code> version of the code base, while API users will be able to use <code>v1</code> or <code>latest</code>.
```
Comment (ccurran01) [inline docs/_includes/api-documentation/v3/versioning-and-release-management.md:14]: There is a note just below here about v1 being due to be deprecated. 

Should this be updated to deprecated in both the V2 and V3 files?

```
<ol class="govuk-list govuk-list--bullet">
  <li>V1 (stable - deprecated  on xx May 2021)</li>
  <li>V2 (stable)</li>
  <li>V3 (upcoming in May 2021)</li>
</ol>
```

**Review by pwright08 — COMMENTED** [review]
Could give the main pages in v3 a quick check with the VSCode spell checker extension? Seems a good point to try catch any obvious ones - wouldn't worry about v2 pages, and just if anything jumps out from a quick scan of what the checker flags

File: docs/_includes/api-documentation/v3/versioning-and-release-management.md (hunk 1 above)
Comment (lukeca362) [reply docs/_includes/api-documentation/v3/versioning-and-release-management.md:14]: Good point. I've added that to v2 and v3 and the date as the 3rd May

File: docs/api-documentation/versions-and-changes/v3/changelog.md [hunk 2]
```diff
@@ -8,8 +8,8 @@ title: API Versions and Changes
 This section lists any significant changes made to this document (and by extension, the API interfaces themselves) introduced by each recent and upcoming future release.
 {: .govuk-body}
 
-Version 1.25.14 (15/04/2021):
-{: .govuk-heading-s #v1-25-14}
+Version 2.10.10 (15/04/2021):
```
[the remark this answers is not in the document]
Comment (lukeca362) [reply docs/api-documentation/versions-and-changes/v3/changelog.md:11]: Good idea. I'll move that to a new folder now

[the remark this answers is not in the document]
Comment (lukeca362) [reply docs/api-documentation/versions-and-changes/v3/changelog.md:11]: I have wiped the versions so that it starts as empty

**Comment by lukeca362** [comment]
I'll rebase the 4 commits into 1 when full code review.

**Comment by lukeca362** [comment]
I will rebase to 1 commit once fully reviewed using `git rebase -i origin/master`

File: docs/_includes/api-documentation/v2/versioning-and-release-management.md [hunk 3]
```diff
@@ -19,9 +19,9 @@ The Street Manager API services are versioned via the URL path, for example <cod
 {: .govuk-heading-m}
 
 <ol class="govuk-list govuk-list--bullet">
-  <li>V1 (stable - due to be deprecated in May 2021)</li>
+  <li>V1 (stable - deprecated  on 3rd May 2021)</li>
```
Comment (ccurran01) [inline docs/_includes/api-documentation/v2/versioning-and-release-management.md:22]:
```suggestion
  <li>V1 (stable - deprecated on 3rd May 2021)</li>
```

**Review by ccurran01 — APPROVED** [review]
Looks good @lukeca362

**Review by pwright08 — APPROVED** [review]

**Comment by lukeca362** [comment]
rebased to resolve conflicts in index.html because v1 link has been removed

File: docs/_includes/api-documentation/v2/versioning-and-release-management.md [hunk 4]
```diff
@@ -19,9 +19,9 @@ The Street Manager API services are versioned via the URL path, for example <cod
 {: .govuk-heading-m}
 
 <ol class="govuk-list govuk-list--bullet">
-  <li>V1 (stable - due to be deprecated in May 2021)</li>
+  <li>V1 (stable - deprecated on 3rd May 2021)</li>
```
Comment (cpaul92) [inline docs/_includes/api-documentation/v2/versioning-and-release-management.md:22]:
```suggestion
  <li>V1 (stable - deprecated on 1st May 2021)</li>
```
Deprecated on the 1st, decommission on the 4th

Comment (lukeca362) [reply docs/_includes/api-documentation/v2/versioning-and-release-management.md]: @cpaul92 I've also changed this in v3 versioning

**Pull request merged**